### ISSUE TITLE
Issue when trying to add indexers

### ISSUE THREAD

**Issue by moarf56** [Jackett/Jackett#5581]
I have an error message when I try to add indexers, like : 
Sharewood
Spacetorrent

The error message is : 

`An error occured while updating this indexer
Specified argument was out of the range of valid values. Parameter name: Unknown operator:
Click here to open an issue on GitHub for this indexer.`

1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
It's good !

2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
Good

3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

`Jackett.Common.ExceptionWithConfigData: Error: Trop de tentatives de connexion. Veuillez réessayer dans secondes secondes.
   at Jackett.Common.Indexers.CardigannIndexer.checkForError(WebClientStringResult loginResult, IList`1 errorBlocks) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 385
   at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 641
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 869
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29`

`System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: Unknown operator: 
   at CloudFlareUtilities.ChallengeSolver.ApplyDecodingStep(Double number, Tuple`2 step)
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at CloudFlareUtilities.ChallengeSolver.ResolveStepGroup(IEnumerable`1 group)
   at System.Linq.Enumerable.SelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at CloudFlareUtilities.ChallengeSolver.DecodeSecretNumber(String challengePageContent, String targetHost)
   at CloudFlareUtilities.ChallengeSolver.Solve(String challengePageContent, String targetHost)
   at CloudFlareUtilities.ClearanceHandler.PassClearance(HttpResponseMessage response, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestLoginAndFollowRedirect(String url, IEnumerable`1 data, String cookies, Boolean returnCookiesFromFirstCall, String redirectUrlOverride, String referer, Boolean accumulateCookies) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 544
   at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 409
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 869
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29`

Jackett Version 0.11.463.0 / with Docker

Any idees ?

THX
-------------------------------

**Comment by garfield69** [comment]
duplicate of #4982 
its due to the new cloudflare check-you-are-not-a-robot test which is tripping up the 3rd party cloudflareutilites that jackett uses to circumvent this check. waiting on provider for new library. 
keep an eye on 4982 for progress.

**Comment by moarf56** [comment]
> 
> 
> duplicate of #4982
> its due to the new cloudflare check-you-are-not-a-robot test which is tripping up the 3rd party cloudflareutilites that jackett uses to circumvent this check. waiting on provider for new library.
> keep an eye on 4982 for progress.

Okay, thx you !!!
